File: exercice2/scala/src/main/scala/fr/devoxx/devops/logs/spark/Spark5.scala
package fr.devoxx.devops.logs.spark

import fr.devoxx.devops.logs.ApacheAccessLog
import org.apache.spark.rdd.RDD
import org.apache.spark.util.StatCounter

/* Statistiques sur la taille des requêtes */
case class Spark5(rdd: RDD[String]) {

  def process: StatCounter = {
    new StatCounter
  }
}

File: exercice2/java/src/main/java/fr/devoxx/devops/logs/spark/Spark5.java
package fr.devoxx.devops.logs.spark;

import fr.devoxx.devops.logs.ApacheAccessLog;
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;
import org.apache.spark.util.StatCounter;
import scala.Tuple4;

import java.io.Serializable;

/* Statistiques sur la taille des requêtes */
public class Spark5 implements Serializable {

    public StatCounter process(JavaRDD<String> rdd) {
        return new StatCounter();
    }

}
